### PR TITLE
Fix Rubocop Rails/NegateInclude issues

### LINKED ISSUE #12338: Rubocop Rails/NegateInclude
Part of #11482 

https://www.rubydoc.info/gems/rubocop-rails/RuboCop/Cop/Rails/NegateInclude

### PR DESCRIPTION
**Pull request by cyrillefr**
#### What? Why?

- Contributes to  #11482
- Closes #12338 

Fixes one of many rubocop issues

- cop class: RuboCop::Cop::Rails::NegateInclude
- idea: replacing `!array.include?(2)` with `array.exclude?(2)`
- `.rubocop_todo.yml` updated

#### What should we test?
Nothing: the rubocop should raise no offenses when checking linters at Integration.

#### Release notes

Changelog Category (reviewers may add a label for the release notes):

- [ ] User facing changes
- [ ] API changes (V0, V1, DFC or Webhook)
- [X] Technical changes only
- [ ] Feature toggled

The title of the pull request will be included in the release notes.

### CODE REVIEW

**Comment by cyrillefr** [comment]
Hello @dacook or @sigmundpetersen ,

could someone relaunch checks, there is a strange error that do not show up locally.
Thanks :)

**Comment by sigmundpetersen** [comment]
Hey @cyrillefr I have run the spec 10 times, consistently failing.
It really looks like the flaky spec #10027 that somehow got consistent as part of this PR 🤷 
Maybe @filipefurtad0 or @openfoodfoundation/core-devs have some insight

**Comment by cyrillefr** [comment]
Really weird this kind of behaviour that arise on my PR (why me :))
I have looked at the code and there was a comment to switch from JS to Capybara.
So, I have changed the code, it works locally. Maybe it going to solve the problem.
And that is a good use case to try to solve flaky # 10027

**Comment by sigmundpetersen** [comment]
Yay, looks good @cyrillefr 🎉 
That's even better, fixing a flaky spec at the same time 😄 
Let's wait for the devs' review 👍

**Comment by cyrillefr** [comment]
So, is this luck or what ?
Maybe something have changed overnight in testing env, or may the code change.

Hello @sigmundpetersen , 
I don't know if what I have done is the real solution, may be some additional testing to be sure ?
Also, there are 2 issues in one PR here, it is a bit untidy.
Should I leave(close) this PR and make a new one to address the flaky and a new tidy one for the rubocop issue ?

**Comment by cyrillefr** [comment]
Ah, @sigmundpetersen , sorry, did not see your reply.

**Comment by sigmundpetersen** [comment]
I think everything is fine @cyrillefr !
We can just close the flaky issue if the devs think this will close it 👍

File: spec/system/admin/order_cycles/complex_editing_multiple_product_pages_spec.rb [hunk 1]
```diff
@@ -36,9 +36,9 @@
     end
 
     it "select all products" do
-      # replace with scroll_to method when upgrading to Capybara >= 3.13.0
       checkbox_id = "order_cycle_incoming_exchange_0_select_all_variants"
-      page.execute_script("document.getElementById('#{checkbox_id}').scrollIntoView()")
+      elmnt = find_field(id: checkbox_id)
+      scroll_to(elmnt, align: :top)
```
Comment (chahmedejaz) [inline spec/system/admin/order_cycles/complex_editing_multiple_product_pages_spec.rb:41]: - That's great to incorporate the latest recommended method.
- By looking at the flaky spec, we are not getting the element by`checkbox_id`
- Like the proposed changes, I think it's worth trying to use the built-in capybara method as opposed to running the JS script for the purpose
- Let's hope for the best now :)

**Review by chahmedejaz — APPROVED** [review]
Great work! 🎉

**Review by rioug — COMMENTED** [review]
Nice one, thanks @cyrillefr :pray:

**Review by rioug — APPROVED** [review]

**Review by mkllnk — APPROVED** [review]
Great, thanks!

I have seen problematic corrections of this cop in the past because it can't know the type of object at run time but these fixes seem all good. :+1: 

I think that we can merge this straight away.

**Pull request merged**